docs/error-api.asciidoc

```diff
@@ -1,11 +1,25 @@
+[[error-api]]
 == Error API
 
+The APM Server exposes an API Endpoint to send error records. 
+Unless you are implementing an agent, you don't need to know about the specifics of this API.
+
 To send error records you need to send a HTTP POST request to APM Server `errors` endpoint. Information pertaining to the error record must be sent as a JSON object to the endpoint.
 
 --
-See <<error-api-examples,Examples>> for sending a request to the error endpoint.
 
-See the schema definitions for how the error payload has to look like:
+Find more information about: 
+
+* <<error-schema-definition>>
+* <<error-api-examples>>
+
+--
+
+[[error-schema-definition]]
+[float]
+=== Schema Definition
+The APM Server uses a JSON Schema for validating the transaction requests. 
```

Comment:
I normally add a blank line after headlines. It makes it easier to read in my opinion.